ContentAwareGenerator.php

```diff
@@ -155,9 +155,9 @@ protected function getBestLocaleRoute(SymfonyRoute $route, $parameters)
      * @param array $parameters which should contain a content field containing
      *                          a RouteReferrersReadInterface object
      *
-     * @return SymfonyRoute the route instance
-     *
      * @throws RouteNotFoundException if no route can be determined
+     *
+     * @return SymfonyRoute the route instance
```

Comment:
I think having consistency is better than what we are used to, so I would vote to keep this fixer. But we can disable it by disabling `phpdoc_order`.